openlibrary/templates/type/edition/view.html

```diff
@@ -79,9 +79,9 @@
         schema_attribute = cond(include_schema, ' itemprop="author"', '')
         authors = cond(work, work and work.get_authors(), edition and edition.get_authors())
         if authors:
-            return ', '.join('<a href="%s"%s>%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) for a in authors)
+            return ', '.join('<a href="%s">%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) for a in authors)
         elif edition.author_names:
-            return ('<span%s>' % (schema_attribute)) + ", ".join(edition.author_names) + "</span>"
+            return ('<span>' % (schema_attribute)) + ", ".join(edition.author_names) + "</span>"
```

Comment:
There is one substitution variable `schema_attribute` so there should also be one `%s`.

Reply:
Okay I will change that and commit in multiple PR but mostly the commits are of indentation so I thought it will be easy for being reviewed 
Thank you for telling me that I will do a multiple PR of this